### PR TITLE
Data schema requirements and flow

### PR DESCRIPTION
**Pull request by mkcomer**
Updates on data requirements and data flow following conversation with Brent

@julioalex-rezende please let me know if I am missing anything or anything else you feel is helpful to add to this doc.

Thanks!

### CODE REVIEW

**Review by julioalex-rezende — COMMENTED** [review]
Very well summed up, MK!

**Review by Aymalla — APPROVED** [review]
LGTM

File: data-reconciliation-app/docs/data-schema-data-flow.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Numerical Results: For example, the forum app computed the mean and standard deviation based on the opinions (data) submitted by the members and returned this as response data .
+    - ID, Mean, Std
+    - unqiue_id_1, 3.2, 0.01
+    - unique_id_2, 4.5, 0.2
+    - unique_id_3, 3.4, 0.1
+- Categorical Results: For LEI data, Results = CSV table of records, thier status (consensus, out of consensus, or undetermined = categorical data) as well as number of folks in the network you are in or out of consensus with. Something along the lines of:
+    - ID, Status, # Members Agreed, # Members did not agree
+    - unqiue_id_1, IN_CONSENSUS, 5, 0
+    - unique_id_2, OUT_OF_CONSENSUS, 3, 2
+    - unique_id_3, NOT_ENOUGH_VOTES, 2, 1
```
[the remark this answers is not in the document]
Comment (mkcomer) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: @julioalex-rezende  great question. By not enough votes, I mean not enough members have submitted data (or votes conceptually) on that particular data record. Does this clear things up? 

When we design the data reporting api, we should speak to Brent to confirm what specific language he would like to use in the data reporting table.

[the remark this answers is not in the document]
Comment (mkcomer) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: @julioalex-rezende  great question. By not enough votes, I mean not enough members have submitted data (or votes conceptually) on that particular data record. Does this clear things up? 

When we design the data reporting api, we should speak to Brent to confirm what specific language he would like to use in the data reporting table.

[the remark this answers is not in the document]
Comment (mkcomer) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: @julioalex-rezende  great question. By not enough votes, I mean not enough members have submitted data (or votes conceptually) on that particular data record. Does this clear things up? 

When we design the data reporting api, we should speak to Brent to confirm what specific language he would like to use in the data reporting table.

[the remark this answers is not in the document]
Comment (mkcomer) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: @julioalex-rezende  great question. By not enough votes, I mean not enough members have submitted data (or votes conceptually) on that particular data record. Does this clear things up? 

When we design the data reporting api, we should speak to Brent to confirm what specific language he would like to use in the data reporting table.

[the remark this answers is not in the document]
Comment (mkcomer) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: @julioalex-rezende to clarify,

If > 80% of members submitted data (voted) on that record AND 
- if 100% agree on record, record status = CONSENSUS
- else, record status = OUT_OF_CONSENSUS

If <80% of members submitted data (unable to vote) on the record, record status = NOT_ENOUGH_VOTES

Is this your understanding as well?

[the remark this answers is not in the document]
Comment (mkcomer) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: @julioalex-rezende to clarify,

If > 80% of members submitted data (voted) on that record AND 
- if 100% agree on record, record status = CONSENSUS
- else, record status = OUT_OF_CONSENSUS

If <80% of members submitted data (unable to vote) on the record, record status = NOT_ENOUGH_VOTES

Is this your understanding as well?

[the remark this answers is not in the document]
Comment (julioalex-rezende) [reply data-reconciliation-app/docs/data-schema-data-flow.md:49]: That was my understanding as well, @mkcomer 
ship it!

**Pull request merged**